build_helpers/startSiteDevServer.sh

```diff
@@ -5,6 +5,7 @@ PATH=$(npm bin):$PATH
 rm -rf ./__site__
 rm -rf ./__site_prerender__
 ./build_helpers/buildAPIDocs.sh
+webpack --config "$PWD/site/webpack-client.config.js"
 webpack --config "$PWD/site/webpack-prerender.config.js"
 ./build_helpers/buildSiteIndexPages.sh
-webpack-dev-server --config "$PWD/site/webpack-client.config.js" --no-info --content-base __site__ --port ${1-8080}
+webpack-dev-server --config "$PWD/site/webpack-client.config.js"
```

Comment:
content base is no longer a parameter to the CLI, but should now be customized inside the webpack config through the `devServer.static` property